packages/camlp5/camlp5.8.00.02/opam

```diff
@@ -28,8 +28,12 @@ bug-reports: "https://github.com/camlp5/camlp5/issues"
 depends: [
   "ocaml" {>= "4.02" & < "4.14.0"}
   "conf-perl"
-  "conf-perl-ipc-system-simple"
-  "conf-perl-string-shellquote"
+  "conf-perl-ipc-system-simple" {
+    os != "macos"
```

Comment:
```suggestion
    os-distribution != "homebrew"
```
And the same below. You have sent the fix for macports in the other pr

Reply:
Fair cop.  I had thought I'd just disable it for macos, but hey, it's worth seeing if tests will pass with macports.  Fair cop.

Reply:
Is that only needed fr tests? If so, the with-test flag was correct and was fixing an incorrect opam file

Reply:
No, these two conf-perl-* packages are used in a script, "mkcamlp5", that is used to build executables.  That script isn't invoked during the build, so once installed, if these conf-perl-* packages aren't available, the script will fail.  For instance the package "GT" uses this script.

Reply:
There might be something wrong with the macos-homebrew machine(s) ?  The test for that platform has sat waiting for resources for a long time -- hours and hours.

Reply:
Yeah. Feel feee to ignore them for now